src/views/AlertDetails/AlertMetadata/index.tsx

```diff
@@ -18,6 +18,8 @@ import styles from './styles.module.css';
 interface ReferenceOutputProps {
     referenceStr?: string;
 }
+// eslint-disable-next-line react-hooks/rules-of-hooks
+const strings = useTranslation(i18n);
```

Comment:
hooks must be used inside of function. 
https://legacy.reactjs.org/docs/hooks-rules.html#only-call-hooks-from-react-functions